<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCodeDialog</class>
 <widget class="QDialog" name="GCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G Code Dialog</string>
  </property>
  <widget class="QPushButton" name="GCodeWriteButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>G Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="whereB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEPostProcessor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mypost.js</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BPostProcessor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>62</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="onRB">
      <property name="text">
       <string>On</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="outsideRB">
      <property name="text">
       <string>Outside</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="insideRB">
      <property name="text">
       <string>Inside</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>69</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="RbCW">
      <property name="text">
       <string>CW</string>
      </property>
      <property name="icon">
       <iconset resource="occqt.qrc">
        <normaloff>:/Resources/cw.png</normaloff>:/Resources/cw.png</iconset>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RbCCW">
      <property name="text">
       <string>CCW</string>
      </property>
      <property name="icon">
       <iconset resource="occqt.qrc">
        <normaloff>:/Resources/ccw.png</normaloff>:/Resources/ccw.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operation List</string>
   </property>
  </widget>
  <widget class="QListView" name="LVContourList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEOperation">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>52</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Safe Z</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LeSafeZ">
      <property name="inputMask">
       <string>50</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>52</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Plunge Z</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LePlungeZ">
      <property name="inputMask">
       <string>0</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>62</y>
     <width>77</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tools</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="toolCBox">
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BAddContour">
      <property name="text">
       <string>Add Contour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BStartPoint">
      <property name="text">
       <string>Start Point</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>GCodeWriteButton</zorder>
  <zorder>lePath</zorder>
  <zorder>whereB</zorder>
  <zorder>LEPostProcessor</zorder>
  <zorder>BPostProcessor</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>LVContourList</zorder>
  <zorder>LEOperation</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="occqt.qrc"/>
 </resources>
 <connections/>
</ui>
